Microsoft.DotNet.Interactive.PowerShell/DollarProfileHelper.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Interactive.PowerShell
+{
+    using System.Management.Automation;
+    using System.Text;
+
+    internal static class DollarProfileHelper
+    {
+        private const string _profileName = "Microsoft.dotnet-interactive_profile.ps1";
+
+        internal static string AllUsersCurrentHost => GetFullProfileFilePath(forCurrentUser: false);
+        internal static string CurrentUserCurrentHost => GetFullProfileFilePath(forCurrentUser: true);
```

Comment:
Maybe better just use internal readonly fields. As properties, `GetFullProfileFilePath` will be called every time the property is used.

Reply:
Changed